src/fe-gtk/maingui.c

```diff
@@ -53,6 +53,10 @@
 #include "sexy-spell-entry.h"
 #include "gtkutil.h"
 
+#ifdef WIN32
```

Comment:
```suggestion
#ifdef G_OS_WIN32
```

Reply:
Sure, I was just basing on some of other defines I found in the project already. If you want besides that, I can look over the code and change the Win32 defs to the glib provided ones.

Reply:
The place `WM_TIMECHANGE` is used is protected by `G_OS_WIN32` so might as well match there.

Don't bother changing others, its just churn.